frontend/.storybook/main.ts

```diff
@@ -20,6 +20,10 @@ const config: StorybookConfig = {
     name: '@storybook/angular',
     options: {},
   },
+  docs: {
+    autodocs: 'tag',
+  },
```

Comment:
That was probably added as part of a wrong rebase conflict resolution and can be removed.